src/test/java/com/neo4j/docker/utils/DatabaseIO.java

```diff
@@ -3,86 +3,111 @@
 import org.junit.jupiter.api.Assertions;
```

Comment:
Sorry the diff here is large but the file was indented with tabs so I let the IDE re-indent the whole thing